### ISSUE TITLE
Settings backup/restore

### ISSUE THREAD

**Issue by f-bone** [Flowwie/flowwie-freecad#8]
Hi there!

Maybe this question should not be here please remove if totally irrelevant.

I watched your first tutorial on youtube where you explain your settings and everything is great but then we get to the color settings where it gets a bit complicated. It it possible to have your settings backup (maybe not your personal one but the one you use for the tutorials) so we can copy it to our installation? I see here its doable: https://forum.freecadweb.org/viewtopic.php?t=37524

thank you and please forgive me if I am being a total noob.

BR,
George

**Comment by BenBE** [comment]
Do I understand your question correct, that you are looking for the settings mentioned at https://www.youtube.com/watch?v=u8otDF_C_fw&t=541s that can be found at https://github.com/Flowwie/flowwie-freecad/tree/master/flowwies-settings to also be exported as a backup/restore friendly file format?

If so, I'd also much appreciate if this could be included in this repo …

**Comment by f-bone** [comment]
yes from what I understand ALL settings can be copied-pasted from one installation to another... so that will include everything and colors.
